examples/console_consumer.v2.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright 2014 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+"use strict";
+
+var KafkaRest = require(".."),
+    argv = require('minimist')(process.argv.slice(2));
+
+    argv = {
+        url: "https://skiff-api.dev.saas.nutanix.com",
```

Comment:
Use generic urls like http://example.com instead of company specific ones. Same for group and topic. test-group and test-topic.